<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="sidebarSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="outlineSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="outlineView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTabWidget" name="editorTabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QTabWidget" name="sidebarTabs">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="toolsTab">
        <attribute name="title">
         <string>Tools</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolBox" name="toolBox">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="tabSpacing">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>312</width>
              <height>183</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>312</width>
              <height>183</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Page 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="helpTab">
        <attribute name="title">
         <string>Documentation</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTextBrowser" name="helpBrowser">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewDocument"/>
    <addaction name="actionOpenDocument"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menu_Format">
    <property name="title">
     <string>&amp;Format</string>
    </property>
    <addaction name="actionParagraphStyle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Format"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDocument"/>
   <addaction name="actionOpenRecent"/>
   <addaction name="actionOpenDocument"/>
   <addaction name="actionSave"/>
   <addaction name="actionRevert"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionRecompute"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QToolBar" name="textToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionTextBold"/>
   <addaction name="actionTextItalic"/>
   <addaction name="actionClearFormatting"/>
   <addaction name="actionTextMath"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
   <addaction name="separator"/>
   <addaction name="actionInsertCAS"/>
   <addaction name="actionInsertGraph"/>
   <addaction name="actionInsertTable"/>
   <addaction name="actionInsertImage"/>
   <addaction name="actionReference"/>
   <addaction name="actionNewLabel"/>
   <addaction name="separator"/>
   <addaction name="actionSpellCheck"/>
  </widget>
  <action name="actionNewDocument">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-new.svg</normaloff>:/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenDocument">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-open.svg</normaloff>:/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/help-about.svg</normaloff>:/icons/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionTextBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-text-bold.svg</normaloff>:/icons/format-text-bold.svg</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionTextItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-text-italic.svg</normaloff>:/icons/format-text-italic.svg</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italic text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-justify-left.svg</normaloff>:/icons/format-justify-left.svg</iconset>
   </property>
   <property name="text">
    <string>Left align</string>
   </property>
   <property name="toolTip">
    <string>Left aligned text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-justify-right.svg</normaloff>:/icons/format-justify-right.svg</iconset>
   </property>
   <property name="text">
    <string>Right align</string>
   </property>
   <property name="toolTip">
    <string>Right aligned text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-justify-center.svg</normaloff>:/icons/format-justify-center.svg</iconset>
   </property>
   <property name="text">
    <string>Center align</string>
   </property>
   <property name="toolTip">
    <string>Centered text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-justify-fill.svg</normaloff>:/icons/format-justify-fill.svg</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
   <property name="toolTip">
    <string>Justified text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionInsertTable">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/insert-table.svg</normaloff>:/icons/insert-table.svg</iconset>
   </property>
   <property name="text">
    <string>Insert table</string>
   </property>
   <property name="toolTip">
    <string>Insert table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionInsertImage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/insert-image.svg</normaloff>:/icons/insert-image.svg</iconset>
   </property>
   <property name="text">
    <string>Insert image</string>
   </property>
   <property name="toolTip">
    <string>Insert image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionClearFormatting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-text-none.svg</normaloff>:/icons/format-text-none.svg</iconset>
   </property>
   <property name="text">
    <string>Clear formatting</string>
   </property>
   <property name="toolTip">
    <string>Clear formatting</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-copy.svg</normaloff>:/icons/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-cut.svg</normaloff>:/icons/edit-cut.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-paste.svg</normaloff>:/icons/edit-paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-revert.svg</normaloff>:/icons/document-revert.svg</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-export.svg</normaloff>:/icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpenRecent">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open-recent.svg</normaloff>:/icons/open-recent.svg</iconset>
   </property>
   <property name="text">
    <string>Open recent</string>
   </property>
   <property name="toolTip">
    <string>Open recent</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-find.svg</normaloff>:/icons/edit-find.svg</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-save.svg</normaloff>:/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-save-as.svg</normaloff>:/icons/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRecompute">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/view-refresh.svg</normaloff>:/icons/view-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>Recompute all</string>
   </property>
   <property name="toolTip">
    <string>Recompute all</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-undo.svg</normaloff>:/icons/edit-undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-redo.svg</normaloff>:/icons/edit-redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open-menu.svg</normaloff>:/icons/open-menu.svg</iconset>
   </property>
   <property name="text">
    <string>Document settings</string>
   </property>
   <property name="toolTip">
    <string>Document settings</string>
   </property>
  </action>
  <action name="actionReference">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/insert-link.svg</normaloff>:/icons/insert-link.svg</iconset>
   </property>
   <property name="text">
    <string>Insert reference</string>
   </property>
   <property name="toolTip">
    <string>Insert reference</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionSpellCheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/check-spelling.svg</normaloff>:/icons/check-spelling.svg</iconset>
   </property>
   <property name="text">
    <string>Check spelling</string>
   </property>
   <property name="toolTip">
    <string>Check spelling</string>
   </property>
  </action>
  <action name="actionInsertCAS">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/insert-cas.svg</normaloff>:/icons/insert-cas.svg</iconset>
   </property>
   <property name="text">
    <string>Insert CAS</string>
   </property>
   <property name="toolTip">
    <string>Insert CAS entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionInsertGraph">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/insert-graph.svg</normaloff>:/icons/insert-graph.svg</iconset>
   </property>
   <property name="text">
    <string>Insert graph</string>
   </property>
   <property name="toolTip">
    <string>Insert graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionParagraphStyle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/paragraph-style.svg</normaloff>:/icons/paragraph-style.svg</iconset>
   </property>
   <property name="text">
    <string>Paragraph style</string>
   </property>
   <property name="toolTip">
    <string>Paragraph style</string>
   </property>
  </action>
  <action name="actionTextMath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/format-text-math.svg</normaloff>:/icons/format-text-math.svg</iconset>
   </property>
   <property name="text">
    <string>Text math</string>
   </property>
   <property name="toolTip">
    <string>Text math</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionParagraphStyleTextBody">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paragraph</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionParagraphStyleTitle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionParagraphStyleSection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Section</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionParagraphStyleSubsection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Subsection</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionParagraphStyleList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>List</string>
   </property>
   <property name="toolTip">
    <string>Itemized list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionParagraphStyleNumberedList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Numbered list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionNewLabel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/label-new.svg</normaloff>:/icons/label-new.svg</iconset>
   </property>
   <property name="text">
    <string>New label</string>
   </property>
   <property name="toolTip">
    <string>New label</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
